### ISSUE TITLE
plot_glass_brain add_markers make not transparent between the hemispheres

### ISSUE THREAD

**Issue by vagechirkov** [nilearn/nilearn#2331]
Hi!
I would like to make the `plot_glass_brain` not transparent for markers between hemispheres. In other words, do not show the markers (`_add_markers`) on the view of the right hemisphere when it is on the left and vice versa. `plot_connectome` has this setting by default.

Many thanks in advance! 
Best regards,
Valerii

**Comment by bthirion** [comment]
Sorry if I misunderstoof, but doesn't the 'lyrz' address your need ?
https://nilearn.github.io/auto_examples/01_plotting/plot_demo_glass_brain_extensive.html#sphx-glr-auto-examples-01-plotting-plot-demo-glass-brain-extensive-py

**Comment by vagechirkov** [comment]
My problem is that I don't want to see the marker on the leftmost picture.

```python
from nilearn import plotting
display = plotting.plot_glass_brain(None, display_mode='lyrz')
display.add_markers([[20,20,20]])
```

<img width="852" alt="image" src="https://user-images.githubusercontent.com/42982039/76259845-8cb44780-6267-11ea-8166-a3841ffaf8a8.png">

**Comment by GaelVaroquaux** [comment]
I agree that the above is suboptimal. Borderline a bug. I would push for having an alpha of .33 on the point in the distant hemisphere. Do you want to open a pull request?

**Comment by vagechirkov** [comment]
Here is the code that does exactly what I want:
https://github.com/nilearn/nilearn/blob/cc22775122dca01beda90626d63018dc4c82de38/nilearn/plotting/displays.py#L396-L420

I would rather have this as an option in `add_markers`. Additionally, I miss text labels for the markers. I would like it to look like this:
![add_marker](https://user-images.githubusercontent.com/42982039/76290146-b5662c80-62bb-11ea-94e7-1dcf1672fa19.png)

I can open a pull request for this if it is useful.

**Comment by GaelVaroquaux** [comment]
Overall looks good!

I think that I would do the code above with numpy array:
<pre>
if self.direction in ('rl'):
     if self.direction == 'r':
             relevant_coords = (xcoords >= 0)
     elif self.direction == 'l':
             relevant_coords = (xcoords <= 0)    
    ...
     marker_color = marker_color[relevant_coords]
</pre>

With regards to the text, I am not sure that I understand well what you want: do you want to add text next to the labels? Maybe this should be another issue?

**Comment by vagechirkov** [comment]
Yes, agree, this a different issue. But you still can see what I mean in the previous plot (text label is just very small).

**Comment by NicolasGensollen** [comment]
Addressed in #2332
